app/change_sets/coin_change_set.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+class CoinChangeSet < ChangeSet
+  delegate :human_readable_type, to: :model
+  apply_workflow(DraftCompleteWorkflow)
+
+  include VisibilityProperty
+  property :department, multiple: false, required: false
+  property :size, multiple: false, required: false
+  property :die_axis, multiple: false, required: false
+  property :weight, multiple: false, required: false
+  property :donor, multiple: false, required: false
```

Comment:
The coins table in the database doesn't have a donor field. There are a few notes about donor in the PrivateInfo and Info fields in the Accessions table, but they're not used exclusively for that. Should we keep this?

Reply:
I saw these fields displayed on Coin show pages, but I agree that I don't see them in the database.  I'll go ahead and remove them for now, and we can figure out how that should be modeled in another ticket.